src/components/sections/two-columns.json

```diff
@@ -0,0 +1,31 @@
+{
+  "collectionName": "components_sections_two_columns",
+  "info": {
+    "displayName": "TwoColumns"
+  },
+  "options": {},
+  "attributes": {
+    "bottomWave": {
```

Comment:
We don't have a bottom wave on FE and it was discussed with Iryna. Let's discuss it with her if we have it in the requirements. In order not to block the PR we can comment out this part until the resolution.